gui_qt/games_registry_worker.py

```
@@ -0,0 +1,53 @@
+
+    def run(self) -> None:
+        result = refresh_registry_projects(
+            self._workspace_root,
+            project_id=self._project_id,
+            refresh_everything=self._refresh_everything,
+            mode=self._mode,
+            on_progress=self._emit_progress,
+            should_cancel=self._should_cancel,
+        )
+        if not isinstance(result, RegistryActionResult):
+            result = RegistryActionResult(False, "刷新失败：未知结果。")
```

Ruff (0.15.20)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)